src/toil/test/batchSystems/batchSystemTest.py

```diff
@@ -288,7 +288,8 @@ def testResultFile(self):
             """
             Tests that the result file name is formatted properly
             """
-            fileName = BatchSystemSupport._getResultsFileName(self._createConfig().jobStore)
+            # noinspection PyUnresolvedReferences
```

Comment:
could also use `assert isinstance(self.batchSystem, BatchSystemSupport` to remove pyCharm warning. Preference?